scalr/helpers.go

```diff
@@ -12,25 +12,47 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
-func GetEnvironmentByName(environmentName string, scalrClient *scalr.Client) (*scalr.Environment, error) {
-	var environment *scalr.Environment
+type GetEnvironmentByNameOptions struct {
+	Name    *string
+	Account *string `json:",omitempty"`
```

Comment:
No need to specify `json:",omitempty"`, this struct will not be serialized to JSON format